src/lib/params-serializer.ts

```diff
@@ -0,0 +1,55 @@
+import type { CustomParamsSerializer } from 'axios'
+
+export const paramsSerializer: CustomParamsSerializer = (params) => {
+  const searchParams = new URLSearchParams()
+
```

Comment:
Should we check for falsy values (null, undefined) before destructuring in the for loop?